packages/react-dom/src/events/DOMEventResponderSystem.js

```diff
@@ -158,16 +158,21 @@ const eventResponderContext: ReactDOMResponderContext = {
     validateResponderContext();
     const childFiber = getClosestInstanceFromNode(childTarget);
     const parentFiber = getClosestInstanceFromNode(parentTarget);
-    const parentAlternateFiber = parentFiber.alternate;
 
-    let node = childFiber;
-    while (node !== null) {
-      if (node === parentFiber || node === parentAlternateFiber) {
-        return true;
+    if (childFiber != null && parentFiber != null) {
```

Comment:
If this function received a `parentTarget` that was an `Element`, but not an element in React's fiber tree, it would throw an error trying to access `parentFiber.alternate`...